src/main/java/com/breskul/bibernate/demo/entity/Person.java

```diff
@@ -0,0 +1,32 @@
+package com.breskul.bibernate.demo.entity;
+
+import com.breskul.bibernate.annotation.Column;
+import com.breskul.bibernate.annotation.Entity;
+import com.breskul.bibernate.annotation.Id;
+import com.breskul.bibernate.annotation.Table;
+
+@Entity
+@Table(name = "persons")
+public class Person {
+    @Id
```

Comment:
The build is failing because `@Id` annotation should have `value` field to be set.